@MCProp/MCProp.m

```diff
@@ -431,161 +431,390 @@ function display(obj)
             y = MCProp.Convert2MCProp(xm);
         end
         function C = subsasgn(A, S, B)
-            switch S(1).type
-                case '()'
-                    A = MCProp(A);
-                    B = MCProp(B);
-                    if A.IsComplex && ~B.IsComplex
-                        B = complex(B);
-                    end
-                    if ~A.IsComplex && B.IsComplex
-                        A = complex(A);
-                    end
-                    ni = numel(S.subs);
-                    na = ndims(A);
-%                    nb = ndims(B);
-                    if ((ni <= 1) || (ni >= na))
-                        if ni == 0
-                            C = A;
+            %   A(I) = B assigns the values of B into the elements of A specified by
```

Comment:
This line is missing:
%SUBSASGN Subscripted assignment.

Reply:
By the way you can just edit the LinProp.m file, then you can open it an text editor replace all `LinProp` by `DistProp` or `MCProp` and save the file as DistProp.m or MCProp.m.